NMWeb/e2e-testcafe/testSrc/profile/userProfile.testc.ts

```diff
@@ -21,25 +18,25 @@ export function userProfileTests() {
       await t.expect(getWindowDocumentLocation()).contains('/profile');
 
       const geoTags = ['Live', 'Work', 'Study', 'Studied', 'Visit', 'homeTown'];
-      const inputFieldOnMapIDNum = _.range(36, 42);
       // Iterate through the geolocation pickers and check map
-      for (let index = 0; index < inputFieldOnMapIDNum.length; index++) {
+      for (let index = 0; index < 6; index++) {
```

Comment:
It no longer relays on autogenerated ids, now t selects input relatively to the open popup